src/modules/author_module.rb

```diff
@@ -0,0 +1,50 @@
+require_relative '../classes/author'
+
+module AuthorModule
+  def find_authors
+    puts 'There are no authors yet, please add an author(s)!' if @authors.empty?
+    puts 'Authors:'
+    @authors.each_with_index do |author, index|
+      puts "[Author #{index + 1}]. First Name : #{author.first_name}, Last Name : #{author.last_name} "
+    end
+  end
+
+  def select_author
+    find_authors
+    print "\nPlease select an author by typing the corresponding number : "
+    author_input = gets.chomp.to_i
+    @authors[author_input.to_i - 1]
+  end
+
+  def create_author()
+    print 'Enter the first name of the author to the game: '
+    firstname = gets.chomp
+    print 'Enter the last name of the author to the game: '
```

Comment:
- [OPTIONAL] When creating the author, you print `'Enter the first name of the author to the game: '` and `'Enter the last name of the author to the game: '` respectively. It should be `'Enter the first name of the author of the book: '` and `'Enter the last name of the author of the book: '`. Kindly replace the word _**game**_ with _**book**_. 😎 💻

Reply:
Fixed. Thanks